fieldservice/models/fsm_category.py

```diff
@@ -16,17 +16,17 @@ class FSMCategory(models.Model):
     company_id = fields.Many2one(
         "res.company",
         string="Company",
-        required=True,
+        required=False,
         index=True,
-        default=lambda self: self.env.user.company_id,
```

Comment:
https://github.com/OCA/field-service/pull/950#discussion_r988585875

Reply:
OK, but then the default should stay

Reply:
No, it is something that should not be there because the field is not necessary.

This is how it is in v14 (https://github.com/OCA/field-service/blob/14.0/fieldservice/models/fsm_category.py#L16-L22) and this is how it is in v15 (https://github.com/Tecnativa/field-service/blob/15.0-mig-fieldservice/fieldservice/models/fsm_category.py#L16-L22), there is no difference.

Reply:
Well, not in v13, but OK, at the end is to get used to.